R/colour-text.r

```diff
@@ -5,9 +5,13 @@ testthat_colours <- list(
   error = crayon::red
 )
 
-colourise <- function(text, as = c("passed", "skipped", "error")) {
+#' @export
+colourise <- function(text, as = c("passed", "skipped", "error"), fg, bg) {
```

Comment:
Actually, this breaks on 

```r
colourise(bg = 'blue', fg = 'red', text = 'blah')
```

since any call to `colourise` that uses a named reference to `text` not in the first argument will have to